environment.php

```diff
@@ -22,7 +22,7 @@
 if (!defined('APPLICATION_VERSION')) {
     // Rules for the versioning
     // {OSS version}-{Cloud release version}-{? SNAPSHOT if it's a dev build}
-    define('APPLICATION_VERSION', '3.3-2019.016-SNAPSHOT');
+    define('APPLICATION_VERSION', '3.4-2019.018-SNAPSHOT');
```

Comment:
Since 017 is going to be 3.4, let's bump this up to 3.5.

```suggestion
    define('APPLICATION_VERSION', '3.5-2019.018-SNAPSHOT');
```